dune-project

```diff
@@ -1,2 +1,20 @@
-(lang dune 1.11)
+(lang dune 2.9)
 (name calendars)
+(version 1.0.0)
+
+(generate_opam_files true)
+(license GPL-2.0-only)
+(authors "Daniel de Rauglaudre")
+(maintainers "Elie Canonici-Merle <elie.canonicimerle@geneanet.org>" "Olivier Pierre <olivier.pierre@geneanet.org>")
+(source (github geneweb/calendars))
+
+(package
+  (name calendars)
+  (synopsis "Convert dates between gregorian/julian/french/hebrew calendars")
+  (description "OCaml library to convert dates between gregorian/julian/french/hebrew calendars. Code is originally from Geneweb.")
+  (tags (moon-phase gregorian julian hebrew french-republican calendar date))
+  (depends
+    (dune (>= "2.9.0"))
```

Comment:
This is not needed, it's already added by dune. See the generated opam file, there's two constraint on dune now. :-)

Reply:
thanks!